core/src/main/java/org/wildfly/extras/creaper/core/online/ManagementProtocol.java

```diff
@@ -7,6 +7,14 @@
     HTTP_REMOTING("http-remoting"),
     /** Used in WildFly. Default port 9993. */
     HTTPS_REMOTING("https-remoting"),
+    /**
+     * HTTP transport is used. Default port 9990
+     * Asynchronous execution is not provided.
+     * If an error occurs (server does not respond 401 or header does not contain WWW-Authenticate field after
+     * first request) {@code IllegalStateException is thrown} (if username and password is provided)
+     * Execution on is allowed with <b>no attachments</b> (i.e. deployment operation).
+     */
```

Comment:
Actually, this text as a whole is a bit hard to read. I'd like something like this better:

> With {@code HTTP}, the management client will not use the native management protocol, but will instead use the HTTP management endpoint. The {@code ModelNode}s will be serialized to JSON. Operation attachments are not supported (yet), so deploying will not work.